specification/gedcom.md

```diff
@@ -516,6 +493,125 @@ An extension tag that is not given a URI in the schema structure is called an **
+
+:::example
+Suppose I have multiple sources that give different ages of the wife at a wedding; however, this specification allows only 1 `MARR`.`WIFE`.`AGE`. An extension could not include multiple `MARR`.`WIFE` nor `MARR`.`WIFE`.`AGE`, but could define a new extension `_AGE`, give it a URL, and provide the following definition of this extension structure type at that URL:
+
+> Alternate age: an age attested by some source, but not accepted by the researcher as the actual age of the individual. If the age is accepted by the researcher, the standard tag `AGE` should be used instead.
+
+This alternate age extension structure could be used as follows:
+
+```gedcom
+1 MARR
+2 WIFE
+3 AGE 27y
+3 _AGE 22y
+```
+:::
+:::
+
+
+A **tagged extension structure** is a structure whose tag is either an undocumented extension tag or a documented extension tag with a URI not defined in this specification.
+This specification allows tagged extension structures as records, in the header, and as substructures of every structure type, all with unbounded cardinality.
+Additional limitations on tagged extension structures are defined by the extension authors.
+
+A tagged extension structure may have any payload type. It is recommended that all tagged extension structures with the same tag should have the same payload type, but for backwards compatibility with extensions authored before 7.0, that limitation is not required.
+
+An **extended-use standard structure** is a structure with a standard tag that is the substructure of either a tagged extension structure or an extended-use standard structure.
+The structure type of an extended-use standard structure is defined by its tag and its superstructure's type.
+If this specification defines a structure type with the same tag and superstructure type, the extended-use standard structure has that type.
+Otherwise, the extended-use standard structure's type is identified by the URI created by concatenating the prefix `g7:` to its tag.
+If neither of those results in a structure type defined in this specification, an extended-use standard structure with this tag is not permitted in this context.
+
+Like relocated standard structures, extended-use standard structures may violate this specifications usual restrictions on superstructure and substructure types, but must abide by all other restrictions on their structure type, including any cardinality constraints.
+
+:::example
+Consider the following example, based on the [GEDCOM-L Addendum to the GEDCOM 5.5.1 specification](https://genealogy.net/GEDCOM/)
+
+```gedcom
+0 @P1@ _LOC
+1 NAME Βυζάντιον
+2 DATE FROM 667 BCE TO 324
+1 NAME Νέα Ῥώμη
+2 DATE 330
+1 _LOC @P2@
+0 @P2@ _LOC
+1 NAME Byzantine Empire
+```
+
+This example uses the following features of the extension system:
+
+- `_LOC` is used both with and without a pointer payload, which is not recommended but is permitted.
```

Comment:
I assume this is referring to its use as a record without a pointer payload, and a non-record tag with a pointer payload.  But why is that not recommended?  Unless I'm misunderstanding the point, gedcom7 even added a new `SNOTE` tag that seems to do the same thing.  Would be helpful to elaborate on _why_ it is not recommended, if you want people to follow the recommendation rather than the normal gedcom precedents.

Reply:
I think it's easier to remove the recommendation text.